src/metrics/include/metrics/metrics.hpp

```diff
@@ -555,14 +554,14 @@ class Registry {
 public:
     explicit Registry() = default;
 
-    Counter counter(const std::string& name) {
-        return Counter{this->_counters[name]};
+    v1::Counter counter(const std::string& name) {
+        return v1::Counter{this->_counters[name]};
     }
-    Timer timer(const std::string& name) {
-        return Timer{this->_timers[name]};
+    v1::Timer timer(const std::string& name) {
+        return v1::Timer{this->_timers[name]};
     }
-    Gauge gauge(const std::string& name) {
-        return Gauge{this->_gauges[name]};
+    v1::Gauge gauge(const std::string& name) {
+        return v1::Gauge{this->_gauges[name]};
```

Comment:
Let's mark these and the associated methods in `ActorContext` as deprecated - see [this SO answer](https://stackoverflow.com/questions/295120/c-mark-as-deprecated).

Reply:
After talking about this on slack with Ryan we decided not to officially deprecate them, as they are still widely used internally and will spam the compile logs.